modules/table/src/main/java/org/apache/ignite/internal/table/distributed/schema/PartitionCommandsMarshaller.java

```diff
@@ -29,7 +29,7 @@ public interface PartitionCommandsMarshaller extends Marshaller {
      * Reads required catalog version from the provided buffer.
      *
      * @param raw Buffer to read from.
-     * @return Catalog version.
+     * @return Catalog version. {@code 0} if version is not required for the given command.
```

Comment:
`-1` is more obvious.

Reply:
I think it must be -1 and I made a mistake